upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -290,13 +290,13 @@ func (c *ApplyClusterCmd) Run() error {
 			}
 
 			l.AddTypes(map[string]interface{}{
-				"persistentDisk":       &gcetasks.PersistentDisk{},
-				"instance":             &gcetasks.Instance{},
-				"instanceTemplate":     &gcetasks.InstanceTemplate{},
-				"network":              &gcetasks.Network{},
-				"managedInstanceGroup": &gcetasks.ManagedInstanceGroup{},
-				"firewallRule":         &gcetasks.FirewallRule{},
-				"ipAddress":            &gcetasks.IPAddress{},
+				"Disk":                 &gcetasks.Disk{},
```

Comment:
I don't think I can express how excited I am to see more tasks being added to GCE. This is so great.